contibuting.md

```diff
@@ -1,22 +1,22 @@
 # कंट्रिब्यूटिंग
 
-नमस्ते! हम रोमांचित हैं कि आप इस प्रोजेक्ट में योगदान करना चाहते हैं। इसे अच्छा बनाए रखने के लिए आपकी मदद जरूरी है। 
+नमस्ते! हम रोमांचित हैं कि आप इस प्रोजेक्ट में योगदान करना चाहते हैं। इसे अच्छा बनाए रखने के लिए आपकी मदद जरूरी है।
 
 यह डॉक्यूमेंट आपकी इस प्रोजेक्ट में योदान करने में सहायता करेगा ।
 
 # पुल्ल रिक्वेस्ट कैसे करें
 
 - इस [issue](https://github.com/gatsbyjs/gatsby-hi/issues/1) पर जाएं और पेज नाम कमेंट करें जिसको आप हिंदी में ट्रांसलेट करना चाहते हैं।
 - रिपॉजिटरी को fork करें।
-- Fork की हुई रिपॉजिटरी तो क्लोन करें (या ऑनलाइन Github एडिटर का इस्तेमाल करें)।
+- Fork की हुई रिपॉजिटरी को क्लोन करें (या ऑनलाइन Github एडिटर का इस्तेमाल करें)।
 - निचे दी गयी कमांड से एक नयी ब्रांच बनाएं।
 ```
 git checkout -b नयी-ब्रांच-का-नाम
 ```
 - जो फाइल आप ट्रांसलेट करना चाहते हैं उस फाइल में जाये और पूरी फाइल को हिंदी में ट्रांसलेट करें।
-- ट्रांसलेशन पूरा करने के बाद, सारे बदलाव को निचे दी गयी कमांड्स से `add`, `commit` और `push` करें।
+- ट्रांसलेशन पूरा करने के बाद, सारे बदलाव को निचे दिये गये कमांड्स से `add`, `commit` और `push` करें।
```

Comment:
I think this should be `दी गयी` commands is "estri-ling" 😛

Reply:
@arshadkazmi42 I think commands are not like 'stri-ling or pu-ling'. Also we can not use Hindi like we are using in our daily life. `दी गयी` it's seems like word that we are using in our daily life. so `दिये गये` it will be better.

Reply:
If we are talking about `दी` and `दिये` its ''stri-ling or pu-ling".
I am still not sure about this. I still feel `दी` is the right word for it. 
@sakshamtaneja21 @SatyaRF Any thoughts on this?

Reply:
As per @arshadkazmi42 and @SatyaRF suggestion, I have to change it as it was.

Reply:
Thank you @kishanjasani and @SatyaRF